Android/app/src/main/java/com/example/goaltracker/EviFrag.kt

```diff
@@ -0,0 +1,5 @@
+import androidx.fragment.app.Fragment
+import com.example.goaltracker.R
+
+class EviFrag:Fragment(R.layout.fragment_evidence) {
```

Comment:
While I haven't seen it in any professional projects, I think Shortening 'Fragment' to 'Frag' works because it's a common term in Android development. But I do not feel the same way about shortening 'Evidence' to 'Evi'

Reply:
Will change them to full form no prob :)